handler/app/app.go

```diff
@@ -73,7 +73,7 @@ func CreateApp(appz core.AppService, appPerUserLimit int) http.HandlerFunc {
 			return
 		}
 
-		if len(appArr) >= appPerUserLimit {
+		if appPerUserLimit != 0 && len(appArr) >= appPerUserLimit {
```

Comment:
Add a new rule: if appPerUserLimit is 0, means no limit. same for botPerUserLimit.